scipy/special/_erfinv.h

```diff
@@ -0,0 +1,62 @@
+#ifndef ERF_INV_H
```

Comment:
Since we've effectively forked Cephes, we've decided that `special/cephes` will be "the place for the C code to go". So I would put this in `cephes/erfinv.c` and then add it to `cephes_names.h`, `cephes.h`, and `_cephes.pxd`.

Reply:
If we are going to move these into ``special/cephes`` it would make sense to group the inverse functions with the original ``erf,erfc`` functions. These are currently defined in ``special/cephes/ndtr.c``. Would it be worth moving all of these (``erf, erfc, erfinv, erfcinv``) into their own file?